src/parcsr_ls/par_mgr_solve.c

```diff
@@ -744,6 +744,16 @@ hypre_MGRCycle( void              *mgr_vdata,
                               U_array[fine_grid]);
                hypre_ParVectorAllZeros(U_array[fine_grid]) = 0;
             }
+            else if ((mgr_data -> level_smoother)[fine_grid])
```

Comment:
See comment above about possible issue if level_smooth_type is not 'unset' if user set specific smoother at this level.

Reply:
Resolved offline